### PR TITLE
Release notes for 7.0.21 and 6.1.44

### PR DESCRIPTION
**Pull request by knisbet**

### CODE REVIEW

File: docs/7.x/changelog.md [hunk 1]
```diff
@@ -51,6 +51,27 @@ extend updates past End of Support through customer agreements if required.
...
 
+### 7.0.21 LTS (October 22nd, 2020)
+
+#### Improvements
+
+* Update TLS cipher suites for Kubernetes components ([#2265](https://github.com/gravitational/gravity/pull/2265), [planet#782](https://github.com/gravitational/planet/pull/782)).
+* Satellite queries for system pods will use less load by searching only the `kube-system` and `monitoring` namespaces for critical pods that aren't running ([#2250](https://github.com/gravitational/gravity/pull/2250), [planet#775](https://github.com/gravitational/planet/pull/775), [satellite#281](https://github.com/gravitational/satellite/pull/281)).
+* Implement a check to prevent installation if the chosen overlay network conflicts with the host networking ([#2204](https://github.com/gravitational/gravity/pull/2204)).
+* Implement a check so that a phase is only rolled back if dependent phases have also been rolled back ([#2219](https://github.com/gravitational/gravity/pull/2219)).
+* Increase base value of expand concurrency to allow a minimum of 4 simaltanous worker joins ([#2205](https://github.com/gravitational/gravity/pull/2205)).
```
Comment (a-palchikov) [inline docs/7.x/changelog.md:62]: simultaneous

File: docs/7.x/changelog.md [hunk 2]
```diff
@@ -51,6 +51,27 @@ extend updates past End of Support through customer agreements if required.
...
+* Include additional cloud metadata as part of `gravity status` output ([#2202](https://github.com/gravitational/gravity/pull/2202)).
+
+#### Bugfixes
+
+* Fix an issue where flannel could corrupt iptables rules if newly generated rules don't exactly match rules previously used ([#2265](https://github.com/gravitational/gravity/pull/2265), [planet#777](https://github.com/gravitational/planet/pull/777), [flannel#7](https://github.com/gravitational/flannel/pull/7)).
+* Fix an issue when using GCE integrations that unneccesary oauth scopes would be requested ([#2265](https://github.com/gravitational/gravity/pull/2265), [planet#777](https://github.com/gravitational/planet/pull/777), [flannel#7](https://github.com/gravitational/flannel/pull/8)).
```
Comment (a-palchikov) [inline docs/7.x/changelog.md:68]: unnecessary

File: docs/7.x/changelog.md [hunk 3]
```diff
@@ -622,6 +643,21 @@ to learn how to gain insight into how the cluster status changes over time.
...
+
+#### Improvements
+
+* Update TLS cipher suites for Kubernetes components ([#2264](https://github.com/gravitational/gravity/pull/2264), [planet#781](https://github.com/gravitational/planet/pull/781)).
+* Satellite queries for system pods will use less load by searching only the `kube-system` and `monitoring` namespaces for critical pods that aren't running ([#2249](https://github.com/gravitational/gravity/pull/2249), [planet#774](https://github.com/gravitational/planet/pull/774), [satellite#280](https://github.com/gravitational/satellite/pull/280)).
+
+#### Bugfixes
+
+* Fix an issue where flannel could corrupt iptables rules if newly generated rules don't exactly match rules previously used ([#2264](https://github.com/gravitational/gravity/pull/2264), [planet#778](https://github.com/gravitational/planet/pull/778), [flannel#7](https://github.com/gravitational/flannel/pull/7)).
+* Fix an issue when using GCE integrations that unneccesary oauth scopes would be requested ([#2264](https://github.com/gravitational/gravity/pull/2264), [planet#778](https://github.com/gravitational/planet/pull/778), [flannel#7](https://github.com/gravitational/flannel/pull/8)).
```
Comment (a-palchikov) [inline docs/7.x/changelog.md:656]: unnecessary
Also, maybe capitalize OAuth?

**Review by a-palchikov — APPROVED** [review]

File: docs/7.x/changelog.md (hunk 1 above)
Comment (UnderGreen) [inline docs/7.x/changelog.md:62]: sim**u**ltaneous

**Review by UnderGreen — APPROVED** [review]

**Pull request merged**